setup.py

```diff
@@ -13,7 +13,8 @@
 ]
 
 extras_require = {
-    ':python_version < "3"': 'py2-ipaddress >= 3.4.1',
+    ':python_version < "3.5"': 'backports.ssl_match_hostname >= 3.5',
+    ':python_version < "3.5"': 'ipaddress',
```

Comment:
According to https://github.com/felixonmars/docker-py/blob/backports/docker/ssladapter/ssladapter.py#L18 and https://bugs.python.org/issue23239, the IP address support wasn't backported to Python 3.3 and 3.4 (yet?), so when I use `< 3.3` the tests failed.